src/components/Filters/__tests__/Company.spec.jsx

```diff
@@ -45,85 +29,18 @@ function setupSnapshot(initialFixture) {
   return renderer.create(
     <Provider store={store}>
       <IntlProvider locale="en">
-        <ReduxCompany />
```

Comment:
🍒  2

This uses the `ReduxCompany` component, which is `<Company />` + the redux bindings `mapDispatch` & `mapState`

This is required for snapshots that put values from the stores in the snapshots